cmake/ctypesgen.cmake

```diff
@@ -40,7 +40,7 @@ endforeach()
 
 set(INC_HEADERS)
 foreach(INCHDR ${INCHDRS})
-  list(APPEND INC_HEADERS "-I\"${INCHDR}\"")
+  list(APPEND INC_HEADERS "-I${INCHDR}")
```

Comment:
I fixed it by quoting in https://github.com/OSGeo/grass/pull/6298 and this PR reverts it?